docs/source/sparse.rst

```diff
@@ -167,6 +171,160 @@ receiving a particular layout. We are working on an API to control the result la
 and recognize it is an important feature to plan a more optimal path of execution for
 any given model.
 
+.. _sparse-semi-structured-docs:
+
+Sparse Semi-Structured Tensors
+++++++++++++++++++++++++++++++
+
+.. warning::
+
+   Sparse semi-sturctured tensors are currently a prototype feature and subject to change. Please feel free to open an issue to report a bug or if you have feedback to share. 
+
+Semi-Structured sparsity is a sparsity layout introduced in NVIDIA's Ampere architecutre. 
```

Comment:
nit: architecture

Please put this text through a spellchecker once it's ready. It's public anyway.